Tryouts/Core/Services/ModulesService/Hosts/BackgroundModuleHost.cs

```diff
@@ -0,0 +1,44 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+
+using MorganStanley.ComposeUI.Tryouts.Core.Abstractions.Modules;
+
+namespace MorganStanley.ComposeUI.Tryouts.Core.Services.ModulesService.Hosts
+{
+    internal class BackgroundModuleHost : ModuleHostBase
+    {
+        private readonly IModuleRunner _runner;
+
+        public BackgroundModuleHost(string name, Guid instanceId, IModuleRunner runner) : base(name, instanceId)
+        {
+            _runner = runner;
+        }
+
+        public override ProcessInfo ProcessInfo => new ProcessInfo(
```

Comment:
Nitpick: if this is a property then I wouldn't instantiate a new object every time it is accessed.

Reply:
This was brought up in the previous PR as well, ProcessInfo was turned into a struct based on that discussion. The reasoning is that the contents of the class can change, but it should be immutable, and changes should be communicated via the LifeCycle events, not by e.g. monitoring it
I'm open for discussion about alternative solutions :)